modules/cli/pages/index.adoc

```diff
@@ -1,4 +1,53 @@
 = Command Line tool for Export
 :description: Explain how to use and configure the CLI to export data from a Bonita database
 
-IMPORTANT: this is a work in progress!!
+== Overview
+The Command Line tool is used to provision data extracted from Bonita instances into a Bonita Process Insights environment for deeper process analysis.
+
+The command line tool is delivered in a package, containing a basic documentation about commands and a sample configuration file.
+
+== Export data
+=== Configuration
+Before exporting, you need to configure the connection to the Bonita Instance from which you want to export the data.
+
+The sample configuration should look like this :
+[source,yaml]
+----
+bonita:
+  database:
+    host: localhost
+    port: 5432
+    name: bonita
+    username: bonita
+    password: bpm
+    jdbc-url: myJdbcUrl
```

Comment:
**question:** can we avoid to duplicate the documentation here and mention that this is available in the documentation embeeded in the distribution?
This risks to be a maintenance issue if we duplicate the content.

Reply:
I remove the configuration sample part to avoid duplication/maintenance changes. I did keep the jdbc url explanation for the different database, should I remove it too?